src/libraries/System.Text.Json/src/System.Text.Json.csproj

```
@@ -12,16 +12,7 @@
     <EnableAOTAnalyzer>true</EnableAOTAnalyzer>
     <PackageDescription>Provides high-performance and low-allocating types that serialize objects to JavaScript Object Notation (JSON) text and deserialize JSON text to objects, with UTF-8 support built-in. Also provides types to read and write JSON text encoded as UTF-8, and to create an in-memory document object model (DOM), that is read-only, for random access of the JSON elements within a structured view of the data.
 
-Commonly Used Types:
-System.Text.Json.JsonSerializer
-System.Text.Json.JsonDocument
-System.Text.Json.JsonElement
-System.Text.Json.Utf8JsonWriter
-System.Text.Json.Utf8JsonReader
-System.Text.Json.Nodes.JsonNode
-System.Text.Json.Nodes.JsonArray
-System.Text.Json.Nodes.JsonObject
-System.Text.Json.Nodes.JsonValue</PackageDescription>
+The System.Text.Json library is built-in as part of the shared framework for .NET Core 3.0+ and .NET 5+. You only need to install it as a package for earlier .NET Core versions or other target frameworks.</PackageDescription>
```

Actually we don't support earlier .NET Core versions. This package only supports .NET 6+, netstandard2.0 and net462+.

---

Can't older Core versions still consume the ns2.0 target though? I might change the wording slightly to say
```suggestion
The System.Text.Json library is built-in as part of the shared framework for .NET Core 3.0+ and .NET 5+. The package can be installed when needing to access more recent versions of the library in older frameworks.</PackageDescription>
```

---

The package explicitly blocks the consumption of out of support .NET versions like net461 and netcoreappX.X (technically netcoreapp3.1 isn't yet out of support but it will close after the release so we decided to not support it anymore).

You can check i.e. the latest available System.Text.Json package which throws a build error on net461 or netcoreappX.X: https://www.nuget.org/packages/System.Text.Json/7.0.0-preview.4.22229.4. That is because of the targets file in it that blocks out of support target frameworks.